readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Anna-Kuzmenko.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Anna-Kuzmenko.github.io/layout_stars/report/html_report/)
```

Comment:
Don't forget to add demo and test report links 

<img width="1033" alt="Screenshot 2023-02-23 at 08 28 57" src="https://user-images.githubusercontent.com/104769168/220834932-d614aa9a-3d88-4185-9142-83c92aeb7840.png">